test/gtest/uct/uct_p2p_test.cc

```diff
@@ -140,8 +141,8 @@ void uct_p2p_test::test_xfer_print(O& os, send_func_t send, size_t length,
 void uct_p2p_test::test_xfer_multi(send_func_t send, size_t min_length,
                                    size_t max_length, direction_t direction) {
 
-    /* Trim at 4GB */
-    max_length = ucs_min(max_length, 4ull * 1124 * 1024 * 1024);
+    /* Trim at 2GB */
+    max_length = ucs_min(max_length, 2ull * 1124 * 1024 * 1024);
```

Comment:
how come `ucs_get_phys_mem_size()` did not detect the correct amount of memory on QEMU? see next line in the code..